src/Components/DishItems/DishMenu.js

```diff
@@ -0,0 +1,72 @@
+import DishCategoryRow from "./DishCategoryRow";
+import "./DishMenu.css";
+import DishRow from "./DishRow";
+
+<<<<<<< HEAD
+<<<<<<< HEAD
+const getDishTemplate = function (items, addToCart) {
+=======
+const getDishTemplate = function (items) {
+>>>>>>> 685276b (Load dish items dynamically)
+=======
+const getDishTemplate = function (items) {
+>>>>>>> 685276b (Load dish items dynamically)
+  const rows = [];
+  Object.keys(items).forEach((category) => {
+    const currItem = items[category];
+    rows.push(
+      <DishCategoryRow
+        key={category}
+        heading={category}
+        subtitle={currItem.length}
+      >
+        {currItem.map((item) => {
```

Comment:
writing map logic in DishCategoryRow could make it simpler instead of rendering children component DishCategoryRow